api/swagger/swaggerDoc.json

```diff
@@ -534,21 +534,15 @@
       }
     },
     "PatronsCreatorResponseModelV02": {
-      "required": [
-        "patron"
-      ],
-      "properties": {
-        "patron": {
-          "$ref": "#/definitions/PatronModelV02"
-        }
-      }
-    },
-    "PatronModelV02": {
       "required": [
         "names",
         "patronType"
       ],
       "properties": {
+        "id": {
```

Comment:
Do we get/want to dictate the Sierra patron id?  Is this the promary key?  Would it be generated, if we didn't indicate it?